api/src/main/java/org/eclipse/microprofile/rest/client/RestClientBuilder.java

```diff
@@ -165,6 +168,43 @@ default RestClientBuilder baseUri(URI uri) {
      */
     RestClientBuilder executorService(ExecutorService executor);
 
+    /**
+     * Specifies the SSL context to use when creating secured transport connections to server endpoints
+     * from web targets created by the client instance that is using this SSL context.
+     *
+     * @param sslContext the ssl context
+     * @return the current builder with ssl context set
```

Comment:
For each of the new methods, there should probably also be a `@throws` javadoc tag to indicate that a NullPointerException should be thrown if any of the params are null.

Reply:
should I also add tests that check that?

Reply:
It wouldn't hurt, but it's up to you - I think that will probably be the behavior for most implementations, since the JAX-RS Client SSL APIs already require it.  My main concern was that the javadocs were consistent with the JAX-RS APIs.

Reply:
If you don't think it's needed, I'd keep it with `@throws` only.